test/libsolidity/syntaxTests/abiEncoder/abi_encode_conversions.sol

```diff
@@ -0,0 +1,30 @@
+interface testInterface {
+    function A(address payable) external;
+    function B(function (string calldata) external) external;
+    function C(function (string memory) external) external;
+    function D(string calldata) external;
+}
+
+contract testContract {
+    function g(string calldata) external {}
+    function h(string memory) external {}
+    string s;
+
+    function main() external view {
+        abi.encodeCall(testInterface.A, (address(0)));
```

Comment:
Can you also add a test the other way around? I.e. encoding `address payable` as an input for an `address` argument?